teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -523,7 +534,340 @@ general oversight. Contributors are already hosting Catalogs in their own organi
 
 ### Automated Testing and Dogfooding
 
-TODO
+In this section, we design the automated test infrastructure and test pipelines setup for both Verified Catalogs and Community Catalogs.
+
+#### Test Pipelines
+
+Following the Catalog Contract defined in [TEP-0115](https://github.com/tektoncd/community/blob/main/teps/0115-tekton-catalog-git-based-versioning.md#organization-contract), the owners need to provide one or more test `pipelines` in the `./{resource-type}/{resource-name}/tests/run.yaml` file of the catalog resource directories to run the integration test for the resource.
```

Comment:
nit: the user is providing a pipeline but the file is called run.yaml?

Reply:
Good point, we use `run.yaml` today but the naming can be improved for sure. Let's call it `pipelineRun.yaml`？

Reply:
Hi @dibyom, I have changed it to `pipelineRun.yaml`. PTAL~